org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/messages/Either.java

```diff
@@ -63,7 +64,17 @@ public boolean isLeft() {
 	public boolean isRight() {
 		return right != null;
 	}
-	
+
+	public <T> T map(Function<L, T> mapLeft, Function <R, T> mapRight) {
```

Comment:
The signature should be `map(Function<? super L, ? extends T> mapLeft, Function<? super R, ? extends T> mapRight)` I suppose. Also the functions should be mandatory and null mappers should be prohibited to avoid surprises.

Reply:
Thanks for the suggestion, I applied it.